### DISCUSSION TITLE
Support for mermaid diagrams

### DISCUSSION THREAD

**Discussion by riceset** [community/community#108622] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.149.0 (122782244)
Device: iPhone 13 (iOS 17.2.1)
</details>

**Comment by iskae** [comment]
Hi @riceset 👋 

Thank you for reaching out ❤️ We already have an internal tracking issue for this 🎉 I have added your request in the trackign issue too 👍  Even though we might not be able to support fully fledged mermaid diagrams, we can try our best to show the content for mermaid diagrams 😊 

Please stay tuned for our future releases 🙇

**Comment by VladimirCreator** [comment]
Duplicates: #11595.
